src/Makefile

```diff
@@ -138,7 +138,8 @@ endif
 #  Intel compiler
 #-----------------------------------------------------------------------------
 ifeq (${FC},ifort)
-    OPTS         += -g -O3 -xHost -ip -assume byterecl -fp-model precise -ftz -init=snan,arrays -traceback
+#    OPTS         += -g -O3 -xHost -ip -assume byterecl -fp-model precise -ftz -init=snan,arrays -traceback
```

Comment:
I also realized the `-init=snan,arrays` flag would broke the CM1 run on NCAR's machine using the Intel compiler. I would suggest to removing this line and keeping the updated version only.

Reply:
This compiler flag should be kept and the runtime error is fixed by George in the latest `gpu-opt` branch.